Packs/StringifyArray/Scripts/StringifyArray/StringifyArray.yml

```diff
@@ -0,0 +1,24 @@
+args:
+- default: true
+  description: The array to join if it has single element
```

Comment:
What do you mean if it has a single element? Can you elaborate on the description?